README.md

```diff
@@ -72,6 +72,8 @@ To load mock black spots, run `python scripts/load_black_spots.py --authz 'Token
 
 To load mock interventions, run `python scripts/load_interventions.py --authz 'Token YOUR_AUTH_TOKEN' /path/to/interventions_sample_pts.geojson`.
 
+To generate black spot inputs, run `python scripts/generate_black_spots_input.py /path/to/roads.shp /path/to/records.csv`.
```

Comment:
I remember with HunchLab having a really hard time keeping all the inputs and outputs straight at first because the output from one thing was the input to another stage, so it might make this friendlier to new users to include "training" in the file name so that it is clear it's making features for machine-learning